### ISSUE TITLE
Error running class transformer

### ISSUE THREAD

**Issue by Redlotus99** [MightyPirates/OpenComputers#1998]
I installed the newest OpenComputers (  OpenComputers-MC1.10.2-1.6.0.3-rc.1.jar)  today and got this class transformer error message when I connected to my server. I have attached both the client and server fml logs.

fml-server-latest.log file from server
https://gist.github.com/Redlotus99/8f01b3ab8fbe42859381113312ead079

fml-client-latest.log file from client
https://gist.github.com/Redlotus99/5e81e1b25f61b677c07cc114ad8d9364

**Comment by TripOptic** [comment]
I too had the same message. Both Client and Server files are provided.

https://gist.github.com/TripOptic/923cdf63fc59868a9b88298c6c280fed

**Comment by Redlotus99** [comment]
@TripOptic Does Open Computers seem to work ok for you? It seems like it's functioning as expected for me.

**Comment by TripOptic** [comment]
@Redlotus99 I assume my game would work just as yours but I simply reverted
to beta 4 till the author has time to reply to us.

**Comment by Redlotus99** [comment]
Gotcha. Beta 4 wasn't accepting power input so I just kept it in my modpack until the power issue was fixed which it seems to be in RC1, but I got the transformer error. Still seems playable though.

**Comment by TripOptic** [comment]
@Redlotus99  Well since you seem to be having good luck with it, I'll switch on back.
I'm having some other bizarre issue with the forge server itself. I cant remove a mod that adds a block or it crashes the server upon /fml confirm, there is no server crash log either.

**Comment by gamax92** [comment]
Normally, another person reporting an "Error running the class transformer" issue tends to be not related but your two issues are the same, same modpack I guess?
Either way, extracted the error messages from the log:

```
[OpenComputers/OpenComputers]: Missing implementations for interface mekanism.api.IMekWrench, skipping injection.
[OpenComputers/BiomesOPlenty]: Failed patching btb.renderLeash, method not found.
```

OpenComputers will happily function just fine with these errors though, so you can go on and use the new version despite.

For future people looking over this, if you don't see the same stuff in your log files please open a separate issue since what's causing errors for you is likely different.

**Comment by Redlotus99** [comment]
Awesome thank you.

**Comment by Vexatos** [comment]
Fixed via #2004. Should not appear anymore in the latest release (1.6.0).
